api/v1beta1/awscluster_conversion.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Kubernetes Authors.
+Copyright 2022 The Kubernetes Authors.
```

Comment:
You haven't touched these files, why change these particular ones? There are MANY others with `Copyright 2021 The Kubernetes Authors`. I think either don't change these or change all of them, right? :)